docs/deployment/Azure/cloud_init_configure.md

```diff
@@ -111,18 +111,37 @@ domain-offset:
   <can also set '*'>: <value0>
 
 repair-manager:
+  cluster_name: <the unique cluster name>
+  domain_name: <a domain name, e.g. dltshub.mydomain.com>
```

Comment:
`domain_name` was changed to `portal_url` in https://github.com/microsoft/DLWorkspace/pull/847/files#diff-dc066a8cecf60f7b4d06a012b30c3349R14

Reply:
I believe the entire section of `repair-manager` can be simplified. Please sync with @debbie-alaine

Reply:
depends on whether you want to allow cross-cluster repair-manager and whether we assume that alert-manager and repair-manager always use the same log in info etc.